src/test/resources/features/complaints/ComplaintsSearch.feature

```diff
@@ -100,75 +87,51 @@ Feature: Complaints Search
-      | Correspondent Full Name           | Sam McTester                         |
-      | Correspondent Postcode            | AB1 2CD                              |
-      | Correspondent Email Address       | SamMcTester@Test.com                 |
-      | Complainant Date Of Birth         | 01/01/2001                           |
-      | Complainant Home Office Reference | Test entry for Home Office Reference |
-
-  @IEDETAndSMCRegression @IEDETComplaints
-  Scenario: User can search for a IEDET complaint case by its case reference
-    Given I am logged into "CS" as user "IEDET_USER"
-    When I create a single "IEDET" case
-    And I search for the case by its case reference
     Then the created case should be the only case visible in the search results
 
 
 #     SMC COMPLAINTS
 
   @IEDETAndSMCRegression @SMCComplaints
-  Scenario Outline: User tests SMC complaint case search criteria
+  Scenario: User tests SMC complaints case search criteria
     Given I am logged into "CS" as user "SMC_USER"
-    When I navigate to the "Search" page
-    And I enter "<infoValue>" into the "<infoType>" search field in the "SMC" search configuration
+    When I generate a "SMC" case to validate search functionality
+    And I navigate to the "Search" page
+    And I enter "Sam McTester" into the "Correspondent Full Name" search field
+    And I enter "AB1 2CD" into the "Correspondent Postcode" search field
+    And I enter "SamMcTester@Test.com" into the "Correspondent Email Address" search field
+    And I enter "01/01/2001" into the "Complainant Date of Birth" search field
+    And I enter "Test entry for HO Reference" into the "Complainant Home Office Reference" search field
+    And I enter "123456789" into the "PSU Reference" search field
+    And I enter the current case reference into the Case Reference field on the search screen
     And I click the search button on the search page
-    Then I check that the search results have the correct "<infoType>"
-    Examples:
-      | infoType                          | infoValue                             |
-      | Correspondent full name           | Sam McTester                          |
-      | Correspondent postcode            | AB1 2CD                               |
-      | Correspondent email address       | SamMcTester@Test.com                  |
-      | Complainant date of birth         | 01/01/2001                            |
-      | Complainant Home Office Reference | Test entry for Home Office Reference  |
-      | PSU Reference                     | 123456789                             |
-
-  @IEDETAndSMCRegression @SMCComplaints
-  Scenario: User can search for a SMC complaint case by its case reference
-    Given I am logged into "CS" as user "SMC_USER"
-    When I create a single "SMC" case
-    And I search for the case by its case reference
     Then the created case should be the only case visible in the search results
 
```

Comment:
This will probably have merge issue with the recently merged branch that moved all SMC scenarios.